docsrc/tools/Program.fs

```diff
@@ -66,16 +66,6 @@ Target.create "Build" (fun _ ->
            )
 )
 
-Target.create "Docs" (fun _ ->
```

Comment:
I guess it's not needed anymore ?

Reply:
Doc template links to github directly. 

See https://fsprojects.github.io/FSharp.Formatting/ links to license/release notes